utilities/examples/md_parser.c

```diff
@@ -0,0 +1,177 @@
+/* Metadata parser example for D457.
```

Comment:
Make we should place it in 
`utilities/md_parser`?
Your call

Reply:
it's too small to have own hierarchy, it's better to have set of small and simple examples.